scripts/purge_messages_db.py

```diff
@@ -0,0 +1,18 @@
+# Purge all Messages from the DB
+# In cleaning out old Messages code, this purges any remaining messages in the database
+import django
+django.setup()
+from sefaria.model import NotificationSet
+
+def purge_messages():
+    count = 0
+    ns = NotificationSet({'type': 'message'})
+    print(f"There are {len(ns)} messages in the database which are about to be purged")
+    for message in ns:
```

Comment:
I dont think its critical here, but in general there is a slightly more performant way to delete a MongoSet of records: 
See `AbstractMongoSet.delete(bulk_delete=True)`

Reply:
Okay great, thank you for pointing that out @edamboritz - very helpful.